source/_patterns/02-organisms/global/site-header.mustache

```diff
@@ -4,11 +4,11 @@
       <a href="#maincontent" class="site-header__skip_to_content__link button button--default">Skip to Content</a>
     </div>
     <a href="{{homePagePath}}" class="site-header__logo_link">
-      <picture class="site-header__logo_link_image">
```

Comment:
Why has this been removed? The [sass that relates to it](https://github.com/elifesciences/pattern-library/blob/d86df0b92b6a8c517da42101500ddd798e85c97a/assets/sass/patterns/organisms/site-header.scss#L63) is still there. Should either remove the sass as well (if there's a good reason to), or don't remove either.

Reply:
Sorry.. Should still be there